issue

Traceback (most recent call last):
  File "/home/runner/work/yarb/yarb/yarb.py", line 242, in <module>
    asyncio.run(main())
  File "/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/runner/work/yarb/yarb/yarb.py", line 239, in main
    await job(args)
  File "/home/runner/work/yarb/yarb/yarb.py", line 218, in job
    await bot.send(bot.parse_results(results))
TypeError: object NoneType can't be used in 'await' expression
Error: Process completed with exit code 1.

--- a/bot.py
+++ b/bot.py
@@ -38,7 +38,7 @@ class feishuBot:
             text_list.append(text.strip())
         return text_list
 
-    def send(self, text_list: list):
+    async def send(self, text_list: list):
         for text in text_list:
             print(f'{len(text)} {text[:50]}...{text[-50:]}')
 
@@ -54,7 +54,7 @@ class feishuBot:
                 console.print('[-] feishuBot 发送失败', style='bold red')
                 print(r.text)
 
-    def send_markdown(self, text):
+    async def send_markdown(self, text):
         # TODO 富文本
         data = {"msg_type": "text", "content": {"text": text}}
         self.send(data)
@@ -81,7 +81,7 @@ class wecomBot:
             text_list.append(text.strip())
         return text_list
 
-    def send(self, text_list: list):
+    async def send(self, text_list: list):
         rates = [Rate(20, Duration.MINUTE)] # 频率限制，20条/分钟
         bucket = InMemoryBucket(rates)
         limiter = Limiter(bucket, max_delay=Duration.MINUTE.value)
@@ -122,7 +122,7 @@ class dingtalkBot:
             text_list.append([feed, text.strip()])
         return text_list
 
-    def send(self, text_list: list):
+    async def send(self, text_list: list):
         rates = [Rate(20, Duration.MINUTE)] # 频率限制，20条/分钟
         bucket = InMemoryBucket(rates)
         limiter = Limiter(bucket, max_delay=Duration.MINUTE.value)
@@ -169,7 +169,7 @@ class qqBot:
             text_list.append(text.strip())
         return text_list
 
-    def send(self, text_list: list):
+    async def send(self, text_list: list):
         rates = [Rate(20, Duration.MINUTE)] # 频率限制，20条/分钟
         bucket = InMemoryBucket(rates)
         limiter = Limiter(bucket, max_delay=Duration.MINUTE.value)
@@ -259,7 +259,7 @@ class mailBot:
         print(text)
         return text
 
-    def send(self, text: str):
+    async def send(self, text: str):
         print(f'{len(text)} {text[:50]}...{text[-50:]}')
 
         msg = MIMEText(text, 'html')

